Tests/AmplitudeTests.m

```diff
@@ -1030,26 +1012,34 @@ - (void)testCustomizedLibrary {
 }
 
 - (void)testCustomizedLibraryWithNilVersion {
+    [self.amplitude cleanUp:@"custom_lib"];
+    
     Amplitude *client = [Amplitude instanceWithName:@"custom_lib"];
+    [client setEventUploadThreshold:1];
     [client initializeApiKey:@"blah"];
+    client.updatingCurrently = NO;
     
     client.libraryName = @"amplitude-unity";
     client.libraryVersion = nil;
     
     [client logEvent:@"test"];
     [client flushQueue];
 
-    NSDictionary *event = [client getLastEventFromInstanceName:@"custom_lib"];
+    NSDictionary *event = [client getLastEventWithInstanceName:@"custom_lib"];
     NSDictionary *targetLibraryValue = @{ @"name" : @"amplitude-unity",
                                           @"version" : kAMPUnknownVersion
     };
     
     NSDictionary *currentLibraryValue = event[@"library"];
     XCTAssertEqualObjects(currentLibraryValue, targetLibraryValue);
+  
 }
 
 - (void)testCustomizedLibraryWithNilLibrary {
-    Amplitude *client = [Amplitude instanceWithName:@"custom_lib"];
+    [self.amplitude cleanUp:@"custom_lib2"];
```

Comment:
Is this check here just to ensure there's no data for this instance name?

Reply:
Oh, that's shouldn't be changed. This should still be instance  custom_lib. The clean up will be removed if we make it able to clean up anything.